tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/xml/XMLAttributeAccessTest.java

```diff
@@ -46,6 +46,12 @@ public void testBasicAttributeAccessSyntax() {
         Assert.assertEquals(result.toString(), "attr-val");
     }
 
+    @Test
+    public void testBasicOptionalAttributeAccessSyntax() {
+        Object result = BRunUtil.invoke(compileResult, "getOptionalElementAttrBasic");
+        Assert.assertEquals(result.toString(), "attr-val");
```

Comment:
Shall we do the assertions in ballerina?

Reply:
I will change